openpype/hosts/max/api/lib.py

```diff
@@ -78,7 +78,13 @@ def read(container) -> dict:
                 value.startswith(JSON_PREFIX):
             with contextlib.suppress(json.JSONDecodeError):
                 value = json.loads(value[len(JSON_PREFIX):])
-        data[key.strip()] = value
+        if key.strip() == "active":
+            if value == "true":
+                data[key.strip()] = True
+            else:
+                data[key.strip()] = False
+        else:
+            data[key.strip()] = value
```

Comment:
You seem to always want to use the stripped key, so do it once.
```suggestion
        key= key.strip()
        
        if key== "active":
            # Special boolean conversion
            value = True if value == "true" else False
        
        data[key] = value
```
Are we sure there are no other boolean data entries that we want to convert? Why is only the `active` key special?

Reply:
I am not quite sure but as far as I checked from the debug, it is only bool value in the dict